qiskit/synthesis/permutation/permutation_utils.py

```diff
@@ -0,0 +1,35 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Utility functions for handling permutations."""
+
+
+def _get_ordered_swap(permutation_in):
```

Comment:
Does it mean that there are at most n swaps? Perhaps it's worth to mention this?

Reply:
Sure, I have improved the description message for ``get_ordered_swaps``.